.webpack/webpack.prod.js

```diff
@@ -1,22 +1,59 @@
 /* global __dirname module */
 
 /*
-This configuration should be used for production installs.
-It is the default webpack configuration.
+Production webpack configuration.
 */
-const path = require('path');
 const webpack = require('webpack');
 const { merge } = require('webpack-merge');
 
-const common = require('./webpack.common');
-const projectRootDir = path.resolve(__dirname, '..');
+const TerserPlugin = require('terser-webpack-plugin');
+const CssMinimizerPlugin = require('css-minimizer-webpack-plugin');
 
-module.exports = merge(common, {
+const base = require('./webpack.base');
+
+module.exports = merge(base('production'), {
+  devtool: false,
   mode: 'production',
   plugins: [
     new webpack.DefinePlugin({
-      __OPENMCT_ROOT_RELATIVE__: '""'
+      __OPENMCT_ROOT_RELATIVE__: '""',
+      'process.env.NODE_ENV': JSON.stringify('production')
```

Comment:
We will continue to avoid runtime environment variables for our webpack configuration